proto/asgt/modelregistry/v1alpha1/model_registry.proto

```diff
@@ -6,43 +6,42 @@ option go_package = "modelregistry";
 
 import "google/protobuf/empty.proto";
 import "asgt/type/model.proto";
+import "asgt/type/dataset.proto";
 
 service ModelRegistry {
-  rpc RegisterModel (RegisterModelRequest) returns (RegisterModelResponse);
-  rpc DeleteModel (DeleteModelRequest) returns (google.protobuf.Empty);
+  rpc RegisterModel (RegisterModelRequest) returns (google.protobuf.Empty);
+//  rpc DeleteModel (DeleteModelRequest) returns (google.protobuf.Empty);
+
+  rpc GetCurrentModel (GetCurrentModelRequest) returns (GetCurrentModelResponse);
```

Comment:
Replacing GetLatestModelVersions ?

Reply:
Yes, because we only ever get one model now (in jester). Before we needed to query all old versions to be able to delete them one by one.